cli/cmd/generate_azure.go

```diff
@@ -0,0 +1,737 @@
+package cmd
+
+import (
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/imdario/mergo"
+	"github.com/spf13/cobra"
+
+	"github.com/lacework/go-sdk/lwgenerate/azure"
+	"github.com/pkg/errors"
+)
+
+var (
+	// Define question text here so they can be reused in testing
+	QuestionAzureEnableConfig = "Enable Azure Config Integration?"
+	QuestionAzureConfigName   = "Specify custom Config integration name: (optional)"
+	QuestionEnableActivityLog = "Enable Azure Activity Log Integration?"
+	QuestionActivityLogName   = "Specify custom Activity Log integration name: (optional"
```

Comment:
Nit: Missing a closing bracket here
```suggestion
	QuestionActivityLogName   = "Specify custom Activity Log integration name: (optional)"
```

Reply:
Done